src/views/CoCurricularTranscript/guestTranscript.tsx

```diff
@@ -0,0 +1,116 @@
+import { MembershipHistory, MembershipHistorySession } from 'services/user';
+import { StudentProfileInfo } from 'services/user';
+import { Class } from 'services/peopleSearch';
+import { Participation } from 'services/membership';
+import { TranscriptItems } from 'services/transcript';
+
+const participation1: Participation = Participation.Member;
+const currentYear: number = new Date().getFullYear();
+const session1: MembershipHistorySession = {
+  MembershipID: 2904,
```

Comment:
I would throw a negative number into here rather than using a random ID that might be taken. That could cause unexpected behavior and confusion later. Perhaps just `-1` if that works?

Reply:
Right! I just changed it.